NumberView.cpp

```diff
@@ -8,16 +8,23 @@
 #include <String.h>
 #include <iostream>
 
+float squareSize = 100;
+
+void SetScale (float scale)
+{
+	squareSize *= scale;
```

Comment:
Are you sure it shouldn't be squareSize = 100 * scale ? What happens if that method is called muliple times?
If applying a factor is really what you mean, then the method should be called ScaleBy instead.

Reply:
Good suggestion! I'll rename the function.
But I would have to leave `squareSize *= scale` as it is, 
because if I changed it to `squareSize *= scale * 100`, scaling the window by a factor of 2 would scale the square tiles ... by 200 (which is definitely not what we want).